### ISSUE TITLE
[AdaptiveCards- Main Body Content]: Labels are not associated with the elements under “AdaptiveCards” main section.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8162]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description
**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open above URL with valid credential.
2. Turn on the Narrator using ctrl+win+enter key.
3. Navigate to 'Android' link under Made For label and observe the issue.

**Actual Result:**
Label is not associated with the elements due to which screen reader does not convey the label name with the links. It is announcing as 'Links Android'.

**Expected Result:**
Labels should be associated with the elements so that screen reader should convey the label information as well and it should announce as 'Made for Android Link'.

**Note:**
Same Issue is repro with NVDA as well.

**User Experience:**
Users who rely on screen reader will not be able to know about the purpose of the control if Label is not associated with the controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B3752B546-30D8-44D7-8221-6FC32EEB3AC8%7D&file=MAS%203.3.2%20%E2%80%93%20Labels%20or%20Instructions.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/208685214-0465d2cf-e433-48b9-b99c-4d76c32cf738.mp4

### Card JSON

```JSON
NA
```
